Dockerfile

```diff
@@ -1,25 +1,24 @@
-FROM node:12.18.2-alpine AS BUILD_IMAGE
+FROM node:12.18.3-alpine AS BUILD_IMAGE
 
 # Create app directory
 WORKDIR /usr/src/app
 
 RUN apk update && apk add git
 
-# Bundle app source
-COPY . .
+COPY package*.json ./
+COPY .git ./.git
```

Comment:
Might need to move this below `RUN npm install --production` for efficient caching of Docker layers.